Jenkinsfile

```
@@ -21,6 +21,7 @@ node('gpu') {
             stage('Tests') {
                 sh """
                     . .venv-$BUILD_NUMBER/bin/activate
+                    flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

This will test everything in the local directory and will include a virtual environment. Also all the relevant code is removed in the `Setup` stage after running `pip install .[tests,docs]`
I would suggest something like
```suggestion
                    flake8 `python -c 'import deeppavlov; print(*deeppavlov.__path__)'` --count --select=E9,F63,F7,F82 --show-source --statistics
```